libnmstate/schemas/operational-state.yaml

```diff
@@ -480,6 +481,30 @@ definitions:
                     type: boolean
                   state:
                     type: string
+  interface-team:
+    rw:
+      properties:
+        type:
+          type: string
+          enum:
+          - team
+        team:
+          type: object
+          properties:
```

Comment:
Just for the record, my preference would be not to define some of the team options if we do not want to define all of them because this might depend too much on the team implementation. Instead moving this information to capabilities seems like a better idea so clients can still check which team features are supported. But this is probably too much now.

Reply:
I will create a card on Jira with that task. I think we should offer a way for the users to check which team features are supported. Would you like to discuss that on the future?